tests/topotests/lib/micronet_cli.py

```diff
@@ -250,6 +250,9 @@ def cli(
     prompt=None,
     background=True,
 ):
+    logging.basicConfig(level=logging.DEBUG, filename="/tmp/topotests/cli-client.log")
```

Comment:
Please don't do this, its running roughshod over the logging that is already setup (and unfortunately quite complex) when cli() is invoked from inside a topotest run.